src/components/academies/AcademyDetailPage.jsx

```diff
@@ -33,6 +37,32 @@ const AcademyDetailPage = () => {
     },
     [config.ALGOLIA_APP_ID, config.ALGOLIA_INDEX_NAME, config.ALGOLIA_SEARCH_API_KEY],
   );
+  useEffect(() => {
+    const fetchPathway = async () => {
+      try {
+        const { hits: pathwayHits, nbHits: nbPathwayHits } = await courseIndex.search('', {
+          filters: `(content_type:learnerpathway) AND academy_uuids:${academyUUID}`,
+          hitsPerPage: 1,
+          page: 0,
+        });
+        if (nbPathwayHits > 0 && pathwayHits[0]?.uuid) {
+          const learnerPathwayUuid = pathwayHits[0].uuid;
+          const learnerPathwayService = new LearnerPathwayService({ learnerPathwayUuid });
+          const data = await learnerPathwayService.fetchLearnerPathwayData();
+          setPathwayData({ title: data.title, overview: data.overview, pathwayUuid: learnerPathwayUuid });
+        } else {
+          setPathwayData({});
+        }
+      } catch (error) {
+        logError(error);
+      }
+    };
+    fetchPathway();
+  }, [academyUUID, courseIndex]);
+
+  function isObjEmpty(obj) {
```

Comment:
Utility functions like this should be placed in the utils file so that they can be utilized by other components too. We already have one.

Reply:
Moved to utils.js